### ISSUE TITLE
EmailSubscription - subscribe button hidden on mobile

### ISSUE THREAD

**Issue by przemokon** [Oksydan/modern-prestashop-starter-theme#100]
On mobile version there is no "subscribe" button. It's hidden via hidden-xs-down class.

```
<input class="btn btn-primary float-right hidden-xs-down" name="submitNewsletter" type="submit" value="Subskrybuj">
```

After removing class ```hidden-xs-down``` button is displaying correctly on mobile.

**Comment by Oksydan** [comment]
Hi @przemokon,

I was able to reproduce an issue. I will inform problem when it will be resolved.

**Comment by Oksydan** [comment]
Hi @przemokon,

Problem has been resolved with [this commit](https://github.com/Oksydan/modern-prestashop-starter-theme/commit/8bfa38ace77a14ee0ed32e0f050ad42fb56489e5). 
I will be released with next version 2.2.0.
